recipes/cssmin/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cssmin" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  sha256: 0649244b431c8d604d443047c66156cf5b0d0677c6beacbdaf40594ee2fd4ea4
+  url: https://github.com/zacharyvoase/cssmin/archive/v{{ version }}.tar.gz
+
+build:
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  number: 0
+  entry_points:
+    - cssmin = cssmin:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  commands:
+    - cssmin --help
+
+about:
+  license_family: BSD
+  license: BSD
+  dev_url: http://github.com/zacharyvoase/cssmin
+  doc_url: http://github.com/zacharyvoase/cssmin
+  home: http://github.com/zacharyvoase/cssmin
+  summary: A Python port of the YUI CSS compression algorithm.
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - csadorf
```

Comment:
@csadorf -- Are you ok being added as a co-maintainer of the `cssmin` recipe on conda-forge?

Reply:
Yes, that's fine with me.